src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -0,0 +1,583 @@
+                  'NOAA',
+                  'Navy',
+                  'Navy Reserve',
+                  'Public Health Service'
+                ]
+              },
+              dateRange: {
+                $ref: '#/definitions/dateRangeAllRequired'
+              }
+            }
+          }
+        },
+        reservesNationalGuardService: {
+          type: 'object',
+          required: ['unitName', 'obligationTermOfServiceDateRange'],
+          properties: {
+            unitName: {
+              type: 'string',
+              maxLength: 256,
+              pattern: "^([a-zA-Z0-9\\-'.#][a-zA-Z0-9\\-'.# ]?)*$"
+            },
+            unitAddress: {
+              $ref: '#/definitions/address'
+            },
+            unitPhone: {
+              $ref: '#/definitions/phone'
+            },
+            obligationTermOfServiceDateRange: {
+              $ref: '#/definitions/dateRangeAllRequired'
+            },
+            title10Activation: {
+              type: 'object',
+              properties: {
+                title10ActivationDate: {
+                  $ref: '#/definitions/date'
+                },
+                anticipatedSeparationDate: {
+                  $ref: '#/definitions/date'
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    servedInCombatZonePost911: {
+      type: 'boolean'
+    },
+    confinements: {
+      type: 'array',
+      minItems: 1,
+      items: {
+        $ref: '#/definitions/dateRangeAllRequired'
+      }
+    },
+    militaryRetiredPayBranch: {
+      type: 'string',
+      enum: serviceBranches
+    },
+    waiveRetirementPay: {
+      type: 'boolean'
+    },
+    separationPayDate: {
+      type: 'string'
+    },
+    separationPayBranch: {
+      type: 'string',
+      enum: serviceBranches
+    },
+    hasTrainingPay: {
+      type: 'boolean'
+    },
+    waiveTrainingPay: {
+      type: 'boolean'
+    },
+    disabilities: {
+      $ref: '#/definitions/disabilities'
+    },
+    newDisabilities: {
+      type: 'array',
+      items: {
+        type: 'object',
+        required: ['condition', 'cause'],
+        properties: {
+          condition: {
+            type: 'string'
+          },
+          cause: {
+            type: 'string',
+            enum: ['NEW', 'SECONDARY', 'WORSENED', 'VA']
+          },
+          primaryDescription: {
+            type: 'string'
+          },
+          causedByDisability: {
+            type: 'string'
+          },
+          causedByDisabilityDescription: {
+            type: 'string'
+          },
+          worsenedDescription: {
+            type: 'string'
+          },
+          worsenedEffects: {
+            type: 'string'
+          },
+          VAMistreatmentDescription: {
+            type: 'string'
+          },
+          VAMistreatmentLocation: {
+            type: 'string'
+          },
+          VAMistreatmentDate: {
+            type: 'string',
+            format: 'date'
```

Comment:
Can you remove this? I have it gone in another PR and if I can get away with it, I'd like to not have to make PR to vets-json-schema to remove it again. :stuck_out_tongue:

Reply:
Did you see @djmunoz1's [latest comment](https://github.com/department-of-veterans-affairs/vets-website/pull/9032#issuecomment-436334490) on that PR?